apps/meteor/client/components/Omnichannel/modals/ForwardChatModal.tsx

```diff
@@ -110,7 +110,7 @@ const ForwardChatModal = ({
 								withTitle
 								filter={departmentsFilter as string}
 								setFilter={setDepartmentsFilter}
-								options={departments.map(({ _id, name }) => ({ value: _id, label: name }))}
+								options={departments as any}
```

Comment:
same question

Reply:
Same. Originally this component was implemented in JS and it was full of bugs due to its flexibility taking props.